frontend/components/product-list/sections/FilterableProductsSection/ProductListFilters/index.tsx

```diff
@@ -181,17 +185,23 @@ function itemKey(index: number, data: ItemData): string {
    return item.name;
 }
 
-function useFilteredFacets() {
+function useFilteredFacets(wikiInfo: WikiInfoEntry[]) {
    const user = useAuthenticatedUser();
    const isProUser = user.data?.discountTier != null;
    const facets = useFacets();
+   const infoNames = React.useMemo(
+      () => new Set(wikiInfo.map((info) => `facet_tags.${info.name}`)),
+      [wikiInfo]
+   );
    const sortedFacets = React.useMemo(() => {
       return facets.slice().sort((a, b) => a.name.localeCompare(b.name));
    }, [facets]);
    const usefulFacets = React.useMemo(() => {
-      const facets = sortedFacets.filter(isUsefulFacet);
+      const facets = sortedFacets
+         .filter(isUsefulFacet)
+         .filter((facet) => !infoNames.has(facet.algoliaName));
       return isProUser ? facets.filter(isAvailableToProUsers) : facets;
-   }, [sortedFacets, isProUser]);
+   }, [sortedFacets, isProUser, wikiInfo]);
```

Comment:
Might be good to do this, so the dependencies are directly indicated:
```suggestion
   }, [sortedFacets, isProUser, infoNames]);
```

Reply:
Good point, but I went with refactoring this whole bit instead.